nifi-assembly/pom.xml

```diff
@@ -519,6 +519,24 @@ language governing permissions and limitations under the License. -->
             <version>2.5.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-cql-nar</artifactId>
```

Comment:
I am not in favor of including cassandra by default in our convenenience binary.  It makes great sense to build a great supportable cassandra interface again but it needs to be really well written and should not be bundled by default until we find out more about usage.

Reply:
For all external system integrations I'm in favor of a (perhaps set of) interface(s) that separate the external system's API/SDK from the NiFi components in order to BYO library so we can try to be future-proof for upcoming versions and such. This is often a lot more difficult and time-consuming than when we used to just accept all contributions that connected to these systems.

Having said that, I don't want to set a negative tone for PR contributions for external system integrations or else everyone will end up with their own private repos and NiFi's bread and butter of universal data distribution will suffer.